### PR TITLE
substituting GetOwnerModel -> GetModel 

### PR DESCRIPTION
**Pull request by RiccardoRossi**
this is a implementing one of the TODOs in #3073

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_processes/spalart_allmaras_turbulence_model.h [hunk 1]
```diff
@@ -105,7 +105,7 @@ class SpalartAllmarasTurbulenceModel : public Process
         bool ReformDofSet,
         unsigned int TimeOrder)
         : mr_model_part(rModelPart),
```
Comment (loumalouomega) [inline applications/FluidDynamicsApplication/custom_processes/spalart_allmaras_turbulence_model.h:107]: Not related with this PR, but the member variables are supposed to be CammelCase

File: kratos/includes/model_part.h [hunk 2]
```diff
@@ -314,7 +314,7 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     void OverwriteSolutionStepData(IndexType SourceSolutionStepIndex, IndexType DestinationSourceSolutionStepIndex);
 
     //this function returns the "Owner" Model
-    Model& GetOwnerModel()
+    Model& GetModel()
     {
         return mrOwnerModel;
```
Comment (loumalouomega) [inline kratos/includes/model_part.h]: Should the member variable name change too?

**Review by loumalouomega — DISMISSED** [review]
Minor comments, let's see what @KratosMultiphysics/technical-committee says

File: applications/FluidDynamicsApplication/custom_processes/spalart_allmaras_turbulence_model.h (hunk 1 above)
Comment (RiccardoRossi) [reply applications/FluidDynamicsApplication/custom_processes/spalart_allmaras_turbulence_model.h:107]: ok, but as you say not related to this PR

**Review by roigcarlo — APPROVED** [review]
:+1:

**Pull request merged**